ibmdbpy/base.py

```diff
@@ -1780,15 +1780,23 @@ def _create_table(self, dataframe, tablename, primary_key=None):
         if column_string[-1] == ',':
             column_string = column_string[:-1]
 
-        create_table = "CREATE TABLE \"%s\" (%s)" % (tablename, column_string)
+        if '.' in tablename:
+            schema = tablename.split('.')[-2]
```

Comment:
This can be shortened to `schema, tablename2 = tablename.split('.')` (since the code doesn't work anyways if there's more than 2 periods in the tablename).

Reply:
Done